<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueEditor</class>
 <widget class="QDialog" name="IssueEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTextEdit" name="descriptionTextEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>descriptionTextEdit</tabstop>
  <tabstop>savePushButton</tabstop>
  <tabstop>cancelPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>IssueEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savePushButton</sender>
   <signal>clicked()</signal>
   <receiver>IssueEditor</receiver>
   <slot>onSavePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>235</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSavePushButtonClicked()</slot>
 </slots>
</ui>
